tests/python/gaia-ui-tests/gaiatest/apps/homescreen/app.py

```diff
@@ -144,6 +145,10 @@ def visible_apps(self):
         return [self.InstalledApp(self.marionette, root_element)
                 for root_element in self.app_elements if root_element.is_displayed()]
 
+    @property
+    def wait_for_homescreen_visible(self):
```

Comment:
Because this isn't a simple accessor (waiting would disqualify it, plus it doesn't return anything) it shouldn't be a property either. Should remove the @property decorator and add the () where you call it below.